publisher/src/main/java/com/pavlovmedia/oss/jaxrs/publisher/impl/swagger/SwaggerConfiguration.java

```diff
@@ -33,20 +30,15 @@
  * @author Shawn Dempsay {@literal <sdempsay@pavlovmedia.com>}
  *
  */
-@Component(immediate=true, metatype=true)
-@Service(SwaggerConfiguration.class)
-@Properties({
-    @Property(name=Publisher.SCAN_IGNORE, value="true", propertyPrivate=true),
-    @Property(name="com.eclipsesource.jaxrs.publish", boolValue=false, propertyPrivate=true),
-    @Property(name="title", label="Title", value="JAX-RS Project"),
-    @Property(name="description", label="Description"),
-    @Property(name="apiVersion", label="API Version", description="The version of this api", value="1.0"),
-    @Property(name="contactName", label="Contact Name"),
-    @Property(name="contactUrl", label="Contact URL"),
-    @Property(name="contactEmail", label="Contact Email"),
-    @Property(name="licenseName", label="License Name", description="Something like 'Apache 2.0'"),
-    @Property(name="licenseUrl", label="License URL")
-})
+@Component(immediate=true,
+    property= {
+        Publisher.SCAN_IGNORE + "=true",
+        "com.eclipsesource.jaxrs.publish=" + false
```

Comment:
You can just put false inside the string

Reply:
Yes, I will change it.